Sources/Views/Pie/PieChartSlice.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  PieChartSlice.swift
+//  Delirium
+//
+//  Created by Jelle Vandebeeck on 23/08/16.
+//  Copyright © 2016 CocoaPods. All rights reserved.
+//
+
+import UIKit
+
+/// The `PieChartSlice` is responsible for a slice that is displayed in the pie chart.
+@objc public protocol PieChartSlice {
```

Comment:
Diu, ik vind `@objc` mooier, dan is duidelijk waarom dat je dat daar zet. Bij `class` is dat niet...